modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,261 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The following procedures use the Bookinfo application to demonstrate different approaches for configuring sidecar injection.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator, created an `{istio}` resource, and the Operator has deployed {istio}.
+* You have created the `IstioCNI` resource, and the Operator has deployed the necessary `IstioCNI` pods.
+* You have created the namespaces that are to be part of the mesh, and they are discoverable by the Istio control plane.
+* Optional: You have deployed the workloads to be included in the mesh. In the following examples, the Bookinfo has been deployed to the `bookinfo` namespace, but sidecar injection (step 5) has not been configured.
+
+[id="ossm-enabling-sidecar-injection-namespace-labels_{context}"]
+== Enabling sidecar injection with namespace labels
+In this example, all workloads within a namespace receive a sidecar proxy injection, making it the best approach when the majority of workloads in the namespace should be included in the mesh.
+
+.Procedure
+
+. Verify the revision name of the {istio} control plane:
++
+[source,terminal]
+----
+$ oc get istiorevision
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME      TYPE    READY   STATUS    IN USE   VERSION   AGE
+default   Local   True    Healthy   False    v1.23.0   4m57s
+----
++
+Since the revision name is default, the default injection labels can be used without referencing the exact revision name.
+
+. Verify that workloads already running in the desired namespace show `1/1` containers as `READY`. This confirms that the pods are running without sidecars.
++
+[source,terminal]
+----
+$ oc get pods -n bookinfo
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+details-v1-65cfcf56f9-gm6v7      1/1     Running   0          4m55s
+productpage-v1-d5789fdfb-8x6bk   1/1     Running   0          4m53s
+ratings-v1-7c9bd4b87f-6v7hg      1/1     Running   0          4m55s
+reviews-v1-6584ddcf65-6wqtw      1/1     Running   0          4m54s
+reviews-v2-6f85cb9b7c-w9l8s      1/1     Running   0          4m54s
+reviews-v3-6f5b775685-mg5n6      1/1     Running   0          4m54s
+----
+
+. Run the following command at the CLI to apply the injection label to the `bookinfo` namespace:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-injection=enabled
+namespace/bookinfo labeled
+----
+
+. Redeploy existing workloads in the `bookinfo` namespace to enable sidecar injection. Use the following command to perform a rolling update of all workloads:
++
+[source,terminal]
+----
+$ oc -n bookinfo rollout restart deployment
```

Comment:
Since this restarts all deployments in the namespace, it makes sense to use the plural form (deployments), since the singular form might leave the reader wondering which deployment we're restarting.
```suggestion
$ oc -n bookinfo rollout restart deployments
```

Reply:
@luksa Done!